### DISCUSSION TITLE
github-runner-scaler on job level

### DISCUSSION THREAD

**Discussion by adityahariyanto** [kedacore/keda#5431] [Ideas]
Hi KEDA community, I would like to express my gratitude for creating this scaler for my self-hosted-runner on GitHub 😸

Is there any plan in the future to have the KEDA scale up runners based on how many jobs are there in the queue, instead of workflows? 

Looking forward to hearing from you guys!

**Comment by JorTurFer** [comment]
> Is there any plan in the future to have the KEDA scale up runners based on how many jobs are there in the queue, instead of workflows?

I'd say that there isn't any API for that, but maybe I'm wrong. Do you know something about this @Eldarrin ?

**Comment by Eldarrin** [comment]
Hi @adityahariyanto @JorTurFer  , actually it *should* be scaling off job count....

https://github.com/kedacore/keda/blob/cdbcb9fe7e77aa4b35624ae9813b00b2c2200d70/pkg/scalers/github_runner_scaler.go#L611

It is supposed to pickup the running "runs", then look into each "run" to find the number of queued "jobs".

Can you share more details and I'll take a look at it.

Cheers
